util/dvsim/dvsim.py

```diff
@@ -660,19 +666,19 @@ def main():
     if args.purge:
         cfg.purge()
 
-    # If --cov-unr is passed, run UNR to generate report for unreachable
-    # exclusion file.
-    if args.cov_unr:
-        cfg.cov_unr()
-        cfg.deploy_objects()
-        sys.exit(0)
-
-    # In simulation mode: if --cov-analyze switch is passed, then run the GUI
-    # tool.
-    if args.cov_analyze:
-        cfg.cov_analyze()
-        cfg.deploy_objects()
-        sys.exit(0)
+    # Run sub-flows (based on switches passed).
+    if cfg.flow == 'sim':
```

Comment:
Is this change related to GUI support? If not, maybe it's clearer to split it out into a separate commit?

Reply:
This is not related to GUI change but it was just cosmetic. Reverted anyway because it does not matter.